src/neuroconv/tools/nwb_helpers/_models/_zarr_models.py

```diff
@@ -134,3 +136,30 @@ def get_data_io_kwargs(self) -> Dict[str, Any]:
             compressor = False
 
         return dict(chunks=self.chunk_shape, filters=filters, compressor=compressor)
+
+
+class ZarrBackendConfiguration(BackendConfiguration):
+    """A model for matching collections of DatasetConfigurations specific to the Zarr backend."""
+
+    backend: Literal["zarr"] = Field(
+        default="zarr", description="The name of the backend used to configure the NWBFile."
+    )
+    data_io_class: Type[ZarrDataIO] = Field(
+        default=ZarrDataIO, description="The DataIO class that is specific to Zarr."
+    )
+    dataset_configurations: Dict[str, ZarrDatasetConfiguration] = Field(
+        description=(
+            "A mapping from object locations to their ZarrDatasetConfiguration specification that contains all "
+            "information for writing the datasets to disk using the Zarr backend."
+        )
+    )
+    number_of_jobs: int = Field(
+        description=(
+            "Number of jobs to use in parallel during write. Negative slicing conforms with the pattern of indexing "
+            " `list(range(total_number_of_cpu))[number_of_jobs]`; for example, `-1` uses all available CPU, `-2` "
+            "uses all except one, etc."
+        ),
+        ge=-psutil.cpu_count(),  # TODO: should we specify logical=False in cpu_count?
+        le=psutil.cpu_count(),
+        default=-2,  # -2 translates to 'all CPU except for one'
```

Comment:
```suggestion
        default=psutil.cpu_count() -1 
```
I would prefer this as the default.